src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -641,7 +599,7 @@ await using (var gzipStream = new GZipStream(section.Body, CompressionMode.Decom
             }
         }
 
-        private async Task DoImportInternalAsync(JsonOperationContext jsonOperationContext, Stream stream, DatabaseSmugglerOptionsServerSide options,
+        public async Task DoImportInternalAsync(JsonOperationContext jsonOperationContext, Stream stream, DatabaseSmugglerOptionsServerSide options,
```

Comment:
this needs to be moved to processor?

Reply:
This one is used in another EP.. So if I move it to `SmugglerHandlerProcessorForImport` I won't be able to call it without an instance